OWNERS.yaml

```diff
@@ -1,13 +1,11 @@
-- cramforce
-- dvoytenko
-- jridgewell
-- "**/validator-*.protoascii": ampproject/wg-caching
-- "**/validator-*.html": ampproject/wg-caching
-- "**/validator-*.out": ampproject/wg-caching
+- "?cramforce"
+- "?dvoytenko"
+- "?jridgewell"
+- "**/validator-*.{protoascii,html,out}": "#ampproject/wg-caching"
 - "*.md": ampproject/wg-outreach
-- ".*, babel.config.js, bundles.config.js, gulpfile.js, renovate.json":
-  - ampproject/wg-infra
-- "package.json, yarn.lock":
+- "{.*,{{babel,bundles}.config,gulpfile}.js,renovate.json}": ampproject/wg-infra
+- "{package.json,yarn.lock}":
   - ampproject/wg-infra
   - ampproject/wg-runtime
   - ampproject/wg-performance
+- "**/OWNERS.yaml": "#rcebulko"
```

Comment:
The idea behind this was that it would be here only temporarily during this transition period where many owners files are being updated as contributors and teams notice which files are out of date. The `#` will tag me on any PR that touches an owners file, so I can verify syntax and semantics of all updates until I finish the syntax check API for Travis to use (which would ideally happen this week but since I'm build-cop this week it depends).

Technically speaking, it means I could theoretically make myself an owner of something (I will not). Practically speaking, I do not have merge access on this repository (so someone else would have to actually review and merge it anyway), the owners bot is non-blocking currently, and I run and maintain the actual owners bot itself, so from a security perspective it seemed moot. I will remove if you'd prefer.

Reply:
The idea behind this was that it would be here only temporarily during this transition period where many owners files are being updated as contributors and teams notice which files are out of date. The `#` will tag me on any PR that touches an owners file, so I can verify syntax and semantics of all updates until I finish the syntax check API for Travis to use (which would ideally happen this week but since I'm build-cop this week it depends).

Technically speaking, it means I could theoretically make myself an owner of something (I will not). Practically speaking, I do not have merge access on this repository (so someone else would have to actually review and merge it anyway), the owners bot is non-blocking currently, and I run and maintain the actual owners bot itself, so from a security perspective it seemed moot. I will remove if you'd prefer.

Reply:
Please paste a version of this comment as a comment into the yaml file.

Reply:
Done :+1:

Reply:
One more thing (sorry), please also create and add here an issue for cleanup.

Reply:
Done! Created #24685 and added a TODO here.